src/main/java/com/nvidia/spark/rapids/jni/RmmSpark.java

```diff
@@ -186,6 +211,29 @@ public static void threadDoneWithShuffle() {
     threadDoneWithShuffle(getCurrentThreadId());
   }
 
+  /**
+   * This should be called as a part of handling any RetryOOM or SplitAndRetryOOM exception.
+   * The order should be something like.
+   * <ol>
+   *   <li>Catch Exception</li>
+   *   <li>Mark any GPU input as spillable, (should have already had contig split called on it)</li>
+   *   <li>call blockUntilReady</li>
+   *   <li>split the input data if SplitAndRetryOOM</li>
```

Comment:
It is in the header above, this line should include `RetryOOM`

Reply:
This is an ordered list. We only want to split the data if we are asked to.